application/core/LSETwigViewRenderer.php

```diff
@@ -668,7 +668,10 @@ private function getAdditionalInfos($aData, $oTemplate)
             if (!empty($aData["sid"]) || LimeExpressionManager::getLEMsurveyId()) {
                 $sid = empty($aData["sid"]) ? LimeExpressionManager::getLEMsurveyId() : $aData["sid"];
                 $language = empty($aData["language"]) ? App()->getLanguage() : $aData["language"];
-                $aData["aSurveyInfo"] = getSurveyInfo($sid, $language);
+                /* Outdated sid in LimeExpressionManager */
```

Comment:
I think this could be on a block on its own, before line 677.
So you have one block for param initialization (current line 666).
Then a new one for initializing  $aData["aSurveyInfo"]...

But this works ok like this, so no need to reorganize

Reply:
> I think this could be on a block on its own, before line 677. So you have one block for param initialization (current line 666). Then a new one for initializing $aData["aSurveyInfo"]...

Yes initialized to an empty array but since we get from function, and function retur false is surveyu don't exist. Then : check if exist to keep the empty array